tools/hls-verifier/src/HlsCoVerification.cpp

```diff
@@ -24,10 +24,10 @@ bool runCoverification(vector<string> args) {
   }
 
   vector<string> temp;
-  for (auto &arg : args) {
+  for (auto &arg : args)
```

Comment:
nit: LLVM guidelines would keep the outer braces here, because it's a multiline body